packages/notifi-react-card/lib/components/subscription/SubscriptionCardV1.tsx

```diff
@@ -114,14 +115,34 @@ export const SubscriptionCardV1: React.FC<SubscriptionCardV1Props> = ({
     targetGroupName: 'Default',
   });
 
+  const {
+    canary: { isActive: canaryIsActive, frontendClient },
+  } = useNotifiClientContext();
+
+  const { isClientInitialized, isClientTokenExpired } = useMemo(() => {
+    let isClientInitialized = false;
+    let isClientTokenExpired = false;
+    if (canaryIsActive && frontendClient.userState) {
+      isClientInitialized = true;
+      isClientTokenExpired = frontendClient.userState.status === 'expired';
+    } else {
+      isClientInitialized = isInitialized;
+      isClientTokenExpired = isTokenExpired;
+    }
+    return { isClientInitialized, isClientTokenExpired };
+  }, [
```

Comment:
```suggestion
  const { isClientInitialized, isClientTokenExpired } = useMemo(() => {
    let isClientInitialized = canaryIsActive && frontendClient.userState) ?? false;
    let isClientTokenExpired = false;
    if (isClientInitialized) {
      isClientTokenExpired = frontendClient.userState.status === 'expired';
    } else {
      isClientInitialized = isInitialized;
      isClientTokenExpired = isTokenExpired;
    }
    
    wdyt?
    return { isClientInitialized, isClientTokenExpired };
  }, [
```

Reply:
ooo, This is a perfect suggestion. One small think I realized when I tried it in action. The typescript has some compile error because that `isClientInitialized` is assigned as `null | boolean` to which the `isInitialized` is not able to be assigned. So I might want to change it to
```suggestion
  const { isClientInitialized, isClientTokenExpired } = useMemo(() => {
    let isClientInitialized =
      !!(canaryIsActive && frontendClient.userState) ?? false;
    let isClientTokenExpired = false;
    if (isClientInitialized) {
      isClientTokenExpired = frontendClient.userState?.status === 'expired';
    } else {
      isClientInitialized = isInitialized;
      isClientTokenExpired = isTokenExpired;
    }

    return { isClientInitialized, isClientTokenExpired };
  }
```

Reply:
Ah ... I think it might not be a good idea. 
By the original approaches, the `isClientInitialized` and `isClientTokenExpired` will be anyway fallback to adopt `useNotifiSubscribe` state even though `isCanary` is set as `true`. I believe we might go for 
```suggestion
 const { isClientInitialized, isClientTokenExpired } = useMemo(() => {
    let isClientInitialized = false;
    let isClientTokenExpired = false;
    if (canaryIsActive) {
      isClientInitialized = !!frontendClient.userState;
      isClientTokenExpired = frontendClient.userState?.status === 'expired';
    } else {
      isClientInitialized = isInitialized;
      isClientTokenExpired = isTokenExpired;
    }
    return { isClientInitialized, isClientTokenExpired };
  }, [
```